Makefile

```diff
@@ -135,6 +135,10 @@ govet:
 	@echo "Running go vet"
 	go vet ./...
 
+shellcheck-test:
```

Comment:
Please also add `shellcheck-test` to the list of targets for `ocs-operator-ci` on line 159. It should go first in the list.

Reply:
Addition to the `ocs-operator-ci` could even be its own commit...